datafusion/src/physical_optimizer/pruning.rs

```diff
@@ -75,6 +77,10 @@ pub trait PruningStatistics {
     /// return the number of containers (e.g. row groups) being
     /// pruned with these statistics
     fn num_containers(&self) -> usize;
+
+    /// return the number of null values for the named column.
+    /// Note: the returned array must contain `num_containers()` rows.
+    fn null_counts(&self, column: &Column) -> Option<ArrayRef>;
```

Comment:
```suggestion

    /// return the number of null values for the named column as an 
    /// `Option<Int64Array>`.
    /// 
    /// Note: the returned array must contain `num_containers()` rows.
    fn null_counts(&self, column: &Column) -> Option<ArrayRef>;
```

I had to look this up to figure out what type this was required

Reply:
Updated.